spec/requests/employee_invitation_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'EmployeeInvitationController', type: :request do
+  let(:user) { create(:user, :admin) }
+  let(:not_listed_user) { create(:user) }
+  let(:company) { create(:company) }
+  let!(:company_user) { create(:company_user, user: user, company: company) }
+  let(:headers) { { Authorization: sign_in(user) } }
+
+  describe 'POST /companies/:company_id/employee_invitations' do
```

Comment:
Missing specs to ensure your service was called (mock), missing specs to ensure the response body of each scenario